.vscode/launch.json

```diff
@@ -88,7 +88,13 @@
         "DEBUG": "composer*",
         "COMPOSER_ENABLE_ONEAUTH": "false"
       },
-      "outputCapture": "std"
+      "outputCapture": "std",
+      "outFiles": [
+        "${workspaceRoot}/Composer/packages/electron-server/build/**/*.js",
+        "${workspaceRoot}/extensions/**/*.js",
+        "${workspaceRoot}/Composer/packages/server/build/**/*.js"
```

Comment:
Does this allow us to put breakpoints in those packages as well?

Reply:
it should -- when I was testing it was working